packages/client/src/util/api.js

```diff
@@ -65,6 +79,22 @@ module.exports = {
     })
   },
 
+  patch(state, path, changes = {}) {
+    // PATCH takes a body, like POST.
+
+    if (state.session.id && !data.sessionID) {
+      data.sessionID = state.session.id
```

Comment:
We should be using `X-Session-ID` everywhere imo.

Reply:
You're free to merge if you want to fix this later, @towerofnix.

Reply:
Agreed, changed that.